### PR TITLE
[incubator/kruise] Add new chart for open-kruise

### PR DESCRIPTION
**Pull request by FillZpp**
Signed-off-by: Siyu Wang <FillZpp.pub@gmail.com>

#### What this PR does / why we need it:

Add [kruise](https://github.com/openkruise/kruise) chart into incubator.

#### Special notes for your reviewer:

Kruise is at the core of the OpenKruise project. It is a set of controllers which extends and complements Kubernetes core controllers on application workload management.

Read the [documents](https://github.com/openkruise/kruise/blob/master/docs/README.md) for further information.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: incubator/kruise/.helmignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+*.swp
+*.bak
+*.tmp
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+# Helm files
+OWNERS
```
Comment (resouer) [inline incubator/kruise/.helmignore]: Add a new line

File: incubator/kruise/templates/manager.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        - name: cert
+          secret:
+            defaultMode: 420
+            secretName: kruise-webhook-server-secret
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kruise-webhook-server-secret
+  namespace: kruise-system
```
Comment (resouer) [inline incubator/kruise/templates/manager.yaml]: new line here

**Comment by resouer** [comment]
/assign @davidkarlsen

Please take a look :-)

File: incubator/kruise/.helmignore (hunk 1 above)
Comment (FillZpp) [reply incubator/kruise/.helmignore]: done

File: incubator/kruise/templates/manager.yaml (hunk 2 above)
Comment (FillZpp) [reply incubator/kruise/templates/manager.yaml]: done

**Comment by maorfr** [comment]
please take a look at helm/helm#6095 (comment)

we strongly encourage people to self host chart repos:
https://github.com/helm/hub/blob/master/Repositories.md

you can even do that using GitHub pages:
https://github.com/helm/hub/blob/master/docs/github.md#github-pages-as-a-repository

**Pull request closed without merging**